### PR TITLE
add new option for CI of the mean + update documentation

### PR DESCRIPTION
**Pull request by vandenman**
Fixes https://github.com/jasp-stats/jasp-issues/issues/2288

Even if approved this should not be merged (as it shouldn't be included in the upcoming hotfix).

### CODE REVIEW

**Review by Kucharssim — APPROVED** [review]

File: inst/help/Descriptives.md [hunk 1]
```diff
@@ -57,6 +56,14 @@ Descriptives allows the user to obtain basic descriptive statistics, histograms
   - Kurtosis: Kurtosis of the distribution of the data points.
   - Shapiro-Wilk test
   - Sum: Sum of the data points.
+- Inference:
+  - S.E. mean: Standard error of the mean.
+  - Confidence interval for the mean:
+    - Width: width of the confidence interval.
+    - Method: How should the confidence interval be computed? By default, we use a `T model`, which yields results identical to a one-sample t-test. Alternative options are a normal model ($\bar{x} \pm z_{95}\times SE$), or `Bootstrap`.
```
[the remark this answers is not in the document]
Comment (shun2wang) [reply inst/help/Descriptives.md:63]: And I'm glad someone finally used Latex in the help doc! 😀

[the remark this answers is not in the document]
Comment (vandenman) [reply inst/help/Descriptives.md:63]: ah... I was under the assumption that we supported the standard markdown syntax that e.g., GitHub also uses. But I guess not :sweat_smile:

[the remark this answers is not in the document]
Comment (shun2wang) [reply inst/help/Descriptives.md:63]: We indeed support markdown syntax but math is different haha, math is not standard basic markdown syntax, GitHub has its extended support?

[the remark this answers is not in the document]
Comment (shun2wang) [reply inst/help/Descriptives.md:63]: Unfortunately, I didn't find a better way, so we had to escape all backslashes.If anyone has any other method please let me know:-)

**Pull request merged**